lib/fluent/plugin/filter_process_ucs_syslog.rb

```diff
@@ -21,7 +21,7 @@ class ProcessUcsSyslog < Filter
     @@tagesRegex = /\[[\w]+\]\[([\w]+)\]\[([-\w]+)\]\[([-\/\w]+)\]/
     @@restartAdaptorRegex = /Adapter (\d)\/(\d)\/\d restarted/
 
-    @@bladeRegex = /sys\/chassis-(\d)\/blade-(\d)/
+    @@bladeRegex = /sys\/chassis-([\d]+)\/blade-(\d)/
```

Comment:
We should update the blade number part of the regex as well to ward off this same issue there

Reply:
I have not noticed any blade that has double digits. but i'll do it just in case